### PR TITLE
Delete plant

### PR DESCRIPTION
**Pull request by LolaMaria**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,14 +3,15 @@ import HomePage from './pages/HomePage';
...
 import styled from 'styled-components';
 import InputPage from './pages/InputPage';
 import { nanoid } from 'nanoid';
-//import { useLocalStorage } from 'usehooks-ts';
+import { useLocalStorage } from 'usehooks-ts';
 
 export default function App() {
-  const [savedPlants, setSavedPlants] = useState(data);
+  const [savedPlants, setSavedPlants] = useLocalStorage('data', data);
+  console.log(savedPlants);
```
Comment (mbosselmann) [inline src/App.js]: You should remove console.log() :)

File: src/App.js [hunk 2]
```diff
@@ -3,14 +3,15 @@ import HomePage from './pages/HomePage';
 import BookmarkPage from './pages/BookmarkPage';
 import Navigation from './components/Navigation.js';
 import data from './components/data.js';
-import { useState } from 'react';
+//import { useState } from 'react';
```
Comment (mbosselmann) [inline src/App.js]: Not needed anymore?  Then you should remove it. :)

File: src/components/ButtonDelete.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+export default function ButtonDelete({ onClick }) {
+  return (
+    <>
```
Comment (mbosselmann) [inline src/components/ButtonDelete.js]: You don't need the fragments here. :)

**Review by mbosselmann — COMMENTED** [review]
Good work! 👍 But I have three remarks you should consider before merging. @macfrei continues my job. :)

File: src/App.js [hunk 4]
```diff
@@ -3,14 +3,15 @@ import HomePage from './pages/HomePage';
...
 import styled from 'styled-components';
 import InputPage from './pages/InputPage';
 import { nanoid } from 'nanoid';
-//import { useLocalStorage } from 'usehooks-ts';
+import { useLocalStorage } from 'usehooks-ts';
 
 export default function App() {
-  const [savedPlants, setSavedPlants] = useState(data);
+  const [savedPlants, setSavedPlants] = useLocalStorage('data', data);
```
Comment (macfrei) [inline src/App.js]: Maybe consider the localStorage key to be something like `plants` or `savedPlants`. `data` is very generic.

File: src/App.js [hunk 5]
```diff
@@ -32,6 +33,7 @@ export default function App() {
             <HomePage
               savedPlants={savedPlants}
               onHandleBookmarkClick={handleBookmarkClick}
+              onDeleteCard={handleDeletePlant}
```
Comment (macfrei) [inline src/App.js]: Just to be consistent with your names 🙂 
```suggestion
              onDeletePlant={handleDeletePlant}
```

File: src/App.js [hunk 6]
```diff
@@ -41,6 +43,7 @@ export default function App() {
             <BookmarkPage
               savedPlants={savedPlants}
               onHandleBookmarkClick={handleBookmarkClick}
+              onDeleteCard={handleDeletePlant}
```
Comment (macfrei) [inline src/App.js]: Same here
```suggestion
              onDeletePlant={handleDeletePlant}
```

File: src/App.js [hunk 7]
```diff
@@ -73,6 +76,9 @@ export default function App() {
 
     navigate('/');
   }
+  function handleDeletePlant(Id) {
+    setSavedPlants(savedPlants.filter(card => card._id !== Id));
+  }
```
Comment (macfrei) [inline src/App.js]: I would recommend to write the parameter without a capital letter. Usually we only start components with a capital letter.
```suggestion
  function handleDeletePlant(id) {
    setSavedPlants(savedPlants.filter(card => card._id !== id));
  }
```

File: src/components/BookmarkButton.js [hunk 8]
```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 export default function BookmarkButton({ onClick, name, isBooked }) {
   return (
     <Button onClick={onClick} isBooked={isBooked}>
-      {(name = 'Save this plant')}
+      {isBooked ? 'Saved Plant' : 'Click to save this plant'}
     </Button>
   );
 }
```
Comment (macfrei) [inline src/components/BookmarkButton.js]: For better readability I would recommend to write `isBookmared` and not a short version `isBooked`. But that is just my personal opinion 🙂

File: src/components/ButtonDelete.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+          height="30"
+          viewBox="0 0 30 32"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M9.0625 3.1875H8.75C8.92188 3.1875 9.0625 3.04688 9.0625 2.875V3.1875H20.9375V2.875C20.9375 3.04688 21.0781 3.1875 21.25 3.1875H20.9375V6H23.75V2.875C23.75 1.49609 22.6289 0.375 21.25 0.375H8.75C7.37109 0.375 6.25 1.49609 6.25 2.875V6H9.0625V3.1875ZM28.75 6H1.25C0.558594 6 0 6.55859 0 7.25V8.5C0 8.67188 0.140625 8.8125 0.3125 8.8125H2.67187L3.63672 29.2422C3.69922 30.5742 4.80078 31.625 6.13281 31.625H23.8672C25.2031 31.625 26.3008 30.5781 26.3633 29.2422L27.3281 8.8125H29.6875C29.8594 8.8125 30 8.67188 30 8.5V7.25C30 6.55859 29.4414 6 28.75 6ZM23.5664 28.8125H6.43359L5.48828 8.8125H24.5117L23.5664 28.8125Z"
+            fill="#4ECB71"
+          />
+        </svg>
```
Comment (macfrei) [inline src/components/ButtonDelete.js]: I would recommend adding a [title](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) to your svg.
```suggestion
        <svg
          width="28"
          height="30"
          viewBox="0 0 30 32"
          fill="none"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>Delete</title>
          <path
            d="M9.0625 3.1875H8.75C8.92188 3.1875 9.0625 3.04688 9.0625 2.875V3.1875H20.9375V2.875C20.9375 3.04688 21.0781 3.1875 21.25 3.1875H20.9375V6H23.75V2.875C23.75 1.49609 22.6289 0.375 21.25 0.375H8.75C7.37109 0.375 6.25 1.49609 6.25 2.875V6H9.0625V3.1875ZM28.75 6H1.25C0.558594 6 0 6.55859 0 7.25V8.5C0 8.67188 0.140625 8.8125 0.3125 8.8125H2.67187L3.63672 29.2422C3.69922 30.5742 4.80078 31.625 6.13281 31.625H23.8672C25.2031 31.625 26.3008 30.5781 26.3633 29.2422L27.3281 8.8125H29.6875C29.8594 8.8125 30 8.67188 30 8.5V7.25C30 6.55859 29.4414 6 28.75 6ZM23.5664 28.8125H6.43359L5.48828 8.8125H24.5117L23.5664 28.8125Z"
            fill="#4ECB71"
          />
        </svg>
```

File: src/components/ButtonEdit.js [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+            fill="#4ECB71"
+          />
+        </svg>
+      </EditButton>
+    </>
+  );
+}
```
Comment (macfrei) [inline src/components/ButtonEdit.js]: Same here with the `title`, fragments and the `onClick`.
```suggestion
export default function ButtonEdit({ onClick }) {
  return (
      <EditButton onClick={onClick}>
        <svg
          width="30"
          height="32"
          viewBox="0 0 35 34"
          fill="none"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>Edit</title>
          <path
            d="M16 2C12.419 1.99777 8.97321 3.36788 6.37146 5.82853C3.76972 8.28917 2.2098 11.6533 2.01255 15.2289C1.8153 18.8045 2.99572 22.3197 5.31101 25.0516C7.62631 27.7835 10.9004 29.5243 14.46 29.916L14.114 31.306C14.066 31.498 14.034 31.69 14.014 31.878C9.9774 31.3684 6.28648 29.3404 3.69193 26.2064C1.09738 23.0724 -0.205949 19.0678 0.0470361 15.007C0.300021 10.9462 2.09032 7.13433 5.0538 4.34658C8.01727 1.55883 11.9314 0.00459491 16 0C19.9388 0.000171356 23.7391 1.45302 26.6736 4.08041C29.608 6.70781 31.4703 10.3252 31.904 14.24C31.2417 14.0428 30.5494 13.9662 29.86 14.014C29.3813 10.6792 27.7164 7.62875 25.1706 5.42209C22.6249 3.21542 19.369 2.00045 16 2V2ZM17.96 26.754L27.62 17.094C28.3218 16.3928 29.2733 15.999 30.2654 15.9994C31.2575 15.9998 32.2088 16.3942 32.91 17.096C33.6112 17.7978 34.005 18.7493 34.0046 19.7414C34.0042 20.7335 33.6098 21.6848 32.908 22.386L23.248 32.044C22.6853 32.607 21.9802 33.0066 21.208 33.2L18.212 33.948C17.9141 34.0222 17.6021 34.0182 17.3062 33.9362C17.0103 33.8542 16.7407 33.6971 16.5235 33.4801C16.3063 33.2631 16.1489 32.9936 16.0667 32.6978C15.9844 32.402 15.98 32.09 16.054 31.792L16.804 28.796C16.9958 28.0232 17.3956 27.3177 17.96 26.756V26.754Z"
            fill="#4ECB71"
          />
        </svg>
      </EditButton>
  );
}
```

File: src/components/PlantCard.js [hunk 11]
```diff
@@ -49,6 +52,11 @@ export default function PlantCard({
         <Info>Additional info:</Info>
         <ListInfo>{info}</ListInfo>
       </List>
+      <ButtonEdit />
+      <ButtonDelete
+        onClick={() => onDeleteCard(id)}
+        aria-label="delete this card"
```
Comment (macfrei) [inline src/components/PlantCard.js]: The aria-label is also not passed as props so this doesn't do anything here.

**Review by macfrei — COMMENTED** [review]
Nice job Lola :) I added a few things you might want to consider. If you have any questions, please don't hesitate to ask me 👍

**Pull request merged**